src/plugins/FileService/utils/date.ts

```diff
@@ -0,0 +1,9 @@
+export function formatDateTime(value: Date, hiddenSeconds = false) {
```

Comment:
`${value.toLocaleDateString()} ${value.toLocaleTimeString()}`

Reply:
need redesign, current design based fixed design width.

@neruthes @OiCkilL

Reply:
The serialization is defined in https://www.figma.com/file/jcrvENZmip6PUR66Jz6ENb/Plugin-File-Service?node-id=33%3A5